tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/xapi_events_all_parsed.yaml

```diff
@@ -49,6 +49,18 @@ columns:
   python_date_format: null
   type: null
   verbose_name: course_key
+- advanced_data_type: null
+  column_name: course_key_short
+  description: "Course key without the course-v1: prefix"
+  expression: "splitByChar(':', splitByChar('/', course_id)[-1])[-1]"
+  extra: {}
+  filterable: true
+  groupby: true
+  is_active: true
+  is_dttm: false
+  python_date_format: null
+  type: null
+  verbose_name: course_key
```

Comment:
@bmtcril Do you think we need to add the `short_course_key` to the table or something?

Reply:
@Ian2012 I can add this (and `course_key`) as a proper table field if you'd prefer? I left this as a computed field because `course_key` is.

Reply:
No, let's leave it as it's. Optimizations can be added later